### PR TITLE
Use Dulwich for auto-update

### PR DESCRIPTION
**Pull request by roeles**
This should work. Tested with SVN and it appears to work fine. After the first checkout `git status` shows normal output and `git remote -v` even shows github as the origin.

### CODE REVIEW

File: AutoUpdate.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#/usr/bin/env python
```
Comment (jaccovanschaik) [inline AutoUpdate.py:1]: If I simply run "python AutoUpdate.py" it does a git clone or pull (as intended) but then simply dumps me in a Python shell. This should probably test for the right number (and type?) of arguments and give a proper error/usage message.

Comment (roeles) [reply AutoUpdate.py:1]: Excellent catch. Please try again with commit ffd0be77acb749cbe24ff14a4b704150018402a0

**Comment by roeles** [comment]
Not at this point, I'm having trouble using VPN.

**Comment by roeles** [comment]
committed

**Comment by jaccovanschaik** [comment]
Yup, seems to work.

**Pull request merged**